python/ray/train/v2/_internal/state/export.py

```diff
@@ -86,8 +92,33 @@ def train_run_attempt_to_proto(attempt: TrainRunAttempt) -> ProtoTrainRunAttempt
     return proto_attempt
 
 
+def _to_proto_dashboard_panel(panel: Panel) -> ProtoTrainRun.DashboardPanel:
+    """Convert Dashboard Panel to protobuf format."""
+    proto_panel = ProtoTrainRun.DashboardPanel(
+        id=str(panel.id),
+        title=panel.title,
+    )
+
+    return proto_panel
+
+
 def train_run_to_proto(run: TrainRun) -> ProtoTrainRun:
     """Convert TrainRun to protobuf format."""
+
+    # Ordered list of panels to be displayed in the train run dashboard
+    train_run_panels = [
+        CONTROLLER_STATE_PANEL,
+        CHECKPOINT_REPORT_TIME_PANEL,
+        CONTROLLER_OPERATION_TIME_PANEL,
+    ]
+    # Ordered list of panels to be displayed in the train worker dashboard
+    train_worker_panels = [CHECKPOINT_REPORT_TIME_PANEL]
+
+    train_run_panels_proto = [_to_proto_dashboard_panel(p) for p in train_run_panels]
+    train_worker_panels_proto = [
+        _to_proto_dashboard_panel(p) for p in train_worker_panels
+    ]
```

Comment:
all of the panel data (like the actual expr to calculate the metric) gets stripped away in this _to_proto step. Maybe call this `_to_proto_dashboard_panel_metadata` instead?

And make it a `ProtoTrainRun.DashboardPanelMetadata`

Reply:
Also, why do we need to ship the list of dashboard panel ids as part of the run data? Is this to be able to selectively show a few panels if you don't want to show all of them?

Reply:
This is the contract (for now) on how to say which panels correspond to which view (run, worker). I can rename to panel!